.github/workflows/test-coverage.yaml

```diff
@@ -6,16 +6,16 @@ on:
   pull_request:
     branches: [main, master]
 
-name: test-coverage
+name: test-coverage.yaml
+
+permissions: read-all
 
 jobs:
   test-coverage:
     runs-on: ubuntu-latest
     env:
-      GITHUB_PAT: ${{ secrets.PRIVATE_ACTIONS_PAT }}
-      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }} 
```

Comment:
This needs to remain.

Reply:
Decided to try without these lines as that is what the template is currently giving us._